pkg/kritis/review/review.go

```diff
@@ -189,7 +188,7 @@ func (r Reviewer) addAttestations(image string, isp v1beta1.ImageSecurityPolicy,
 			errMsgs = append(errMsgs, err.Error())
 		}
 		// Create Attestation Signature
-		if _, err := r.client.CreateAttestationOccurence(n, image, s); err != nil {
+		if _, err := r.client.CreateAttestationOccurence(n, image, s, &a); err != nil {
```

Comment:
We already get the note in L181, so no need to pass 'a' here.

Reply:
Just pass the project name instead.  For now let's just use grafeas.DefaultProject ("kritis").  Please see my comment in metadata.go.